model/mcp/registry.yaml

```diff
@@ -0,0 +1,181 @@
+groups:
+  - id: registry.mcp
+    type: attribute_group
+    brief: >
+      [Model Context Protocol (MCP)](https://spec.modelcontextprotocol.io) attributes
+    attributes:
+      - id: mcp.method.name
+        brief: The name of the request or notification method.
+        stability: development
+        type:
+          members:
+            - id: notifications_cancelled
+              value: notifications/cancelled
+              brief: >
+                Notification cancelling a previously-issued request.
+              stability: development
+            - id: initialize
+              value: initialize
+              brief: >
+                Request to initialize the MCP client.
+              stability: development
+            - id: notifications_initialized
+              value: notifications/initialized
+              brief: >
+                Notification indicating that the MCP client has been initialized.
+              stability: development
+            - id: notifications_progress
+              value: notifications/progress
+              brief: >
+                Notification indicating the progress for a long-running operation.
+              stability: development
+            - id: ping
+              value: ping
+              brief: >
```

Comment:
Would you mind also updating the `brief` for `ping`? It doesn't provide any progres updates, it just expects an empty payload in response.